src/AppInstallerCommonCore/pch.h

```diff
@@ -12,7 +12,8 @@
 
 #include "TraceLogging.h"
 
-#include <yaml-cpp/yaml.h>
+#define YAML_DECLARE_STATIC
```

Comment:
>YAML_DECLARE_STATIC [](start = 8, length = 19)

Is this intended here?

Reply:
Yes.  I could put it in the project, but I don't see the value.  Here it is tied to the header it actually is related to.

---
In reply to: [468976197](https://github.com/Microsoft/pkg/pull/535#discussion_r468976197) [](ancestors = 468976197)